### ISSUE TITLE
Environment file stay locked after closing root-db.

### ISSUE THREAD

**Issue by yurko7** [DoctorEvidence/lmdb-store#85]
After inserting some amount of data to DB and closing it, file stay locked.
```typescript
import {unlinkSync} from "fs"
import * as lmdb from "lmdb-store"

const dbFile = "./lm.db"
const db = lmdb.open({path: dbFile})

await db.transactionAsync(() => {
    for (let i = 0; i < 5000; i++) {
        db.put(i, {data: i * 2})
    }
})

db.close()
unlinkSync(dbFile)
```
Got an `Error: EPERM: operation not permitted, unlink './lm.db'`

lmdb-store - v1.6.6
node - v14.15.0

**Comment by kriszyp** [comment]
Should be fixed in v1.6.7.

**Comment by yurko7** [comment]
Works like a charm 👍
Thank you!
